lib/genome/online_updaters/docm/updater.rb

```diff
@@ -0,0 +1,132 @@
+require 'net/http'
```

Comment:
This doesn't need to be here - nothing in this module uses `net/http`. You could put it at the top of the `ApiClient` class as it actually uses `Net::HTTP` but its not technically required either place as rails already requires it.

Also this is totally my mistake, I did the same in the CIViC importer; I'll change that :)